stdlib_tests/test_math.py

```diff
@@ -0,0 +1,3 @@
+import math
+
+print(math.sqrt(4))
```

Comment:
You should include the remaining tests that are found in tests/stdlib/test_math.py, namely 12.34, -3.5 ... 'Hello'.

Reply:
Added few more testcases in fc1917f8a717a980507ed8a2ada8f6b0b7753b98.
Since this is only a smoke test, it tests only basic functionality and does not have to be exhaustive.